.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - run: uv pip freeze
 
-      - run: uv run coverage run -m pytest
+      - run: uv run coverage run -m pytest --junitxml=junit.xml -o junit_family=legacy
```

Comment:
Just wondering: why can't Codecov read the "normal" coverage output file and needs a junitxml output?